config/environments/production.rb

```diff
@@ -24,6 +24,8 @@
   config.public_file_server.enabled = ENV['RAILS_SERVE_STATIC_FILES'].present?
 
   # Compress JavaScripts and CSS.
+  # NOTE: If you want to use ES6 in your JS assets, switch to the following:
+  # config.assets.js_compressor = Uglifier.new(harmony: true)
```

Comment:
@citizen428 @janklimo hey should we make this the default? I've noticed some people don't know what is ES6 and what isn't since we use it so frequently, and using ES6 without this setting triggers very confusing errors in heroku deployments. Any reason this is opt in instead of opt out?

Reply:
Agreed with you @hiattp I'd imagine you typically learn the hard way this is off by default

Reply:
It's opt-in because this is not officially supported by Rails, but something I dug up from some GH discussion. But sure, we can enable it ¯\_(ツ)_/¯